### DISCUSSION TITLE
Outages to the Monsoon post-processor, MASS, and XCS

### DISCUSSION THREAD

**Discussion by mo-mahammedk** [MetOffice/simulation-systems#197] [Monsoon]
There will be three separate pieces of maintenance coming up (21st, 22nd, and 28th May). Please see below for details:
 
**Tuesday 21st May 09:00-11:00 - CHG0111202 – Monsoon Post-processor server (exppostproc01) -**
Monthly security patching of the post-processor server will mean it is unavailable whilst it is rebooted (approximately twenty minutes during the maintenance window).
 
**Wednesday 22nd May 10:00-12:00 - CHG0116922 - MASS Planned outage -**
On Wednesday 22nd May there will be an outage to MASS in order to apply RHEL security patching on the MASS servers. Between 10:00 and 12:00 all MOOSE 'PUT', 'GET', 'SELECT' and 'FILTER' commands will fail.
 
**Tuesday 28th May 09:00-16:00 - CHG0117102 - Complete outage to XCS -** 
HPE have raised a change (CHG0117102) which will require downtime on XCS. Cabined 0-0 has a number of failed rectifiers and these are normally replaced with a single cabinet outage. Unfortunately in this instance Cabinet 0-0 contains the boot node for the system and will therefore require down time on the full system.
We'll place a system wide reservation on XCS for the duration of the change, and remove this as soon as HPE have completed their maintenance.
 
As always we will post all service status updates here.

**Comment by mo-mahammedk** [comment]
**Tuesday Monsoon Post-processor update:** Patches have been applied and back from reboot.

**Comment by mo-mahammedk** [comment]
**Wednesday MASS update:** At around 12:07pm on Wednesday, the system was back processing commands.

**Comment by mo-roger-milton** [comment]
**Tuesday XCS maintenance:** Well, if you haven't already worked it out for yourself, XCS has been back for normal work from about 11:30 today. If you encounter any problems, please report them through the usual channels.
